parallelTests.cmake

```diff
@@ -78,6 +78,14 @@ add_test_compare_parallel_simulation(CASENAME spe3
                                      REL_TOL ${coarse_rel_tol_parallel}
                                      TEST_ARGS --linear-solver-reduction=1e-7 --tolerance-cnv=5e-6 --tolerance-mb=1e-8 --tolerance-wells=1e-7)
 
+add_test_compare_parallel_simulation(CASENAME spe3
```

Comment:
this needs a different casename because the casename is used for folder names and it now clashes with the other test. then you also need to use DIR.